apps/system/js/app_chrome.js

```diff
@@ -749,9 +749,10 @@
     };
 
   AppChrome.prototype.setThemeColor = function ac_setThemColor(color) {
-    // Do not set theme color for private windows
+    // Overwrite color for private windows
     if (this.app.isPrivateBrowser()) {
-      return;
+      color = '#392E54';
```

Comment:
Is this to prevent the theme-color from taking effect? I'm ok if we want to do that, please update the comment to say so if that's the reason.